velox/expression/BooleanMix.cpp

```diff
@@ -109,11 +109,7 @@ BooleanMix getFlatBool(
       activeRows.applyToSelected([&](int32_t i) {
         auto index = indices[i];
         bool isNull = nulls && bits::isBitNull(nulls, i);
-        if (mergeNullsToValues && nulls) {
```

Comment:
@waitinfuture Would you clarify why this code is redundant?

Reply:
Hi @mbasmanova , let x=`mergeNullsToValues && nulls`, y = `!isNull && bits::isBitSet(values, index)`, IIUC the following table stands:
| x        | y           | action  |
| ------------- |-------------| -----|
| true      | true| call `bits::setBit(valuesToSet, i);` |
| true      | false       |   not call `bits::setBit(valuesToSet, i);` |
| false | true     |   call `bits::setBit(valuesToSet, i);` |
| false | false     |  not call `bits::setBit(valuesToSet, i);` |

so whether to call `bits::setBit(valuesToSet, i);` only depends on y and is irrelevant to x.

Reply:
Got it. Thank you for explaining.